easybuild/easyconfigs/p/Pysam/Pysam-0.22.1-GCC-13.3.0.eb

```diff
@@ -13,12 +13,9 @@ source_urls = ['https://github.com/pysam-developers/pysam/archive/']
 sources = ['v%(version)s.tar.gz']
 checksums = ['e4981524d7627c53fa0d3f8cbec2bd65c2ea7520092f25e1029af12cb7b82ff6']
 
-builddependencies = [
-    ('Cython', '3.0.10')
-]
-
 dependencies = [
     ('Python', '3.12.3'),
+    ('Cython', '3.0.10'),
```

Comment:
this should be a build dependency from 2024a on.
if pairtools depends on it, then you should add it there too.